File: RedisAPI/Client/Proizvod.js
export class Proizvod {
    constructor(id, Naziv, Cena, Opis, KategorijaId, BrPorudzbina) {
        this.id = id;
        this.naziv = Naziv;
        this.cena = Cena;
        this.opis = Opis;
        this.kategorijaId = KategorijaId;
        this.brPorudzbina = BrPorudzbina;
    }
}

File: RedisAPI/Models/Proizvod.cs
using Redis.OM.Modeling;

namespace Redis.OM.Skeleton.Model;


[Document(StorageType = StorageType.Json, Prefixes = new []{"Proizvod"})]
public class Proizvod
{
    [RedisIdField] [Indexed]public string Id { get; set; }

    [Indexed] public string Naziv {get; set;}

    [Indexed] public int Cena {get; set;}

    [Indexed] public string Opis {get; set;}
    [Indexed] public string KategorijaId {get; set;}
    [Indexed] public int BrPorudzbina {get; set;}
 
}